github/pulls_reviewers.go

```diff
@@ -12,8 +12,8 @@ import (
 
 // ReviewersRequest specifies users and teams for a pull request review request.
 type ReviewersRequest struct {
-	Reviewers     []string `json:"reviewers,omitempty"`
-	TeamReviewers []string `json:"team_reviewers,omitempty"`
```

Comment:
The error message indicates that `Reviewers` is required. Is it also the case that `TeamReviewers` is required?
My intuition would be `TeamReviewers` would not be required because it's a new parameter that most requests won't have.